packages/1155-contracts/src/nft/ZoraCreator1155Impl.sol

```diff
@@ -553,63 +554,8 @@ contract ZoraCreator1155Impl is
         return super.supportsInterface(interfaceId) || interfaceId == type(IZoraCreator1155).interfaceId || ERC1155Upgradeable.supportsInterface(interfaceId);
     }
 
-    function _handleSupplyRoyalty(uint256 tokenId, uint256 mintAmount, bytes memory data) internal returns (uint256 totalRoyaltyMints) {
-        uint256 royaltyMintSchedule = royalties[tokenId].royaltyMintSchedule;
-        if (royaltyMintSchedule == 0) {
-            royaltyMintSchedule = royalties[CONTRACT_BASE_ID].royaltyMintSchedule;
-        }
-        if (royaltyMintSchedule == 0) {
-            // If we still have no schedule, return 0 supply royalty.
-            return 0;
-        }
-        uint256 maxSupply = tokens[tokenId].maxSupply;
-        uint256 totalMinted = tokens[tokenId].totalMinted;
-
-        totalRoyaltyMints = (mintAmount + (totalMinted % royaltyMintSchedule)) / (royaltyMintSchedule - 1);
-        totalRoyaltyMints = MathUpgradeable.min(totalRoyaltyMints, maxSupply - (mintAmount + totalMinted));
-        if (totalRoyaltyMints > 0) {
-            address royaltyRecipient = royalties[tokenId].royaltyRecipient;
-            if (royaltyRecipient == address(0)) {
-                royaltyRecipient = royalties[CONTRACT_BASE_ID].royaltyRecipient;
-            }
-            // If we have no recipient set, return 0 supply royalty.
-            if (royaltyRecipient == address(0)) {
-                return 0;
-            }
-            super._mint(royaltyRecipient, tokenId, totalRoyaltyMints, data);
-        }
-    }
-
     /// Generic 1155 function overrides ///
 
-    /// @notice Mint function that 1) checks quantity and 2) handles supply royalty 3) keeps track of allowed tokens
-    /// @param to to mint to
-    /// @param id token id to mint
-    /// @param amount of tokens to mint
-    /// @param data as specified by 1155 standard
-    function _mint(address to, uint256 id, uint256 amount, bytes memory data) internal virtual override {
-        uint256 supplyRoyaltyMints = _handleSupplyRoyalty(id, amount, data);
-        _requireCanMintQuantity(id, amount + supplyRoyaltyMints);
-
-        super._mint(to, id, amount, data);
-        tokens[id].totalMinted += amount + supplyRoyaltyMints;
```

Comment:
do we still want to track `totalMinted` ? are we using that anywhere?

Reply:
ah good catch - @iainnash wdyt?

Reply:
i just checked our frontend repo and they're using it - i'll add that back

Reply:
yep we want to track this still.